src/uu/chroot/src/chroot.rs

```diff
@@ -97,11 +119,11 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
     }
 }
 
-fn set_context(root: &Path, options: &getopts::Matches) {
-    let userspec_str = options.opt_str("userspec");
-    let user_str = options.opt_str("user").unwrap_or_default();
-    let group_str = options.opt_str("group").unwrap_or_default();
-    let groups_str = options.opt_str("groups").unwrap_or_default();
+fn set_context(root: &Path, options: &clap::ArgMatches) {
+    let userspec_str = options.value_of("userspec");
+    let user_str = options.value_of("user").unwrap_or_default();
+    let group_str = options.value_of("group").unwrap_or_default();
+    let groups_str = options.value_of("groups").unwrap_or_default();
```

Comment:
would be nice to have a test to have proper coverage
(even if this is checking that it is failing)